apps/fsl/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
```

Comment:
```suggestion

set -e

```

Reply:
otherwise, failing to build the image won't produce a failure.

Reply:
Indeed, i need to put that by default for every image, i'll remember it at one point ^^'